drivers/i2c/Kconfig.imx

```diff
@@ -0,0 +1,40 @@
+# Kconfig.mcux - IMX I2C configuration options
+#
+# Copyright (c) 2018, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig IMX_I2C
```

Comment:
Don't add a new Kconfig file, please model this off of what is done for i2c_mcux, use CONFIG_I2C_1, CONFIG_I2C_2, etc.

Reply:
For imx we have the hardware mapped from I2C_1 to I2C_4 and in the Kconfig we have I2C_0 to I2C_3.

What is your suggestion, add the I2C_4 option to Kconfig or use 0 to 3 and consider 0 as 1, 1 as 2 and so on, for the imx?

Reply:
I'm saying add I2C_4 to Kconfig, and just use I2C_1..I2C_4